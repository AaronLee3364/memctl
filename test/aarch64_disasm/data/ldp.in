fffffff0070c0000
a9417bfd a8c24ff4 29402408 a8402428
69cb4586
