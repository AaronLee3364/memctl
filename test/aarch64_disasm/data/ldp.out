fffffff0070c0000  LDP     X29, X30, [SP, #16]
fffffff0070c0004  LDP     X20, X19, [SP], #32
fffffff0070c0008  LDP     W8, W9, [X0]
fffffff0070c000c  LDNP    X8, X9, [X1]
fffffff0070c0010  LDPSW   X6, X17, [X12, #88]!
